.github/workflows/publish-docs.yml

```diff
@@ -8,20 +8,15 @@ on:
     tags:
         - v*
 
+concurrency:
+  group: gh-pages
+
 jobs:
   update-gh-pages:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
 
-    - name: Turnstyle
-      uses: softprops/turnstyle@v1
-      with:
-        abort-after-seconds: 600
-        same-branch-only: false
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
     - name: Fetch gh-pages
       run: git fetch --no-tags --prune --depth=1 origin refs/heads/gh-pages:refs/heads/gh-pages
 
```

Comment:
since we are here, should we perhaps replace this by https://github.com/actions/checkout ?

Reply:
That's an independent/unrelated change. Let's have a separate PR

Reply:
ok